components/commons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spectrum-css/commons",
-  "version": "3.0.6",
+  "version": "4.0.0",
```

Comment:
No need to increment this manually. Lerna will take care of this for us automatically when we do the release.